game-core/src/main/java/games/strategy/triplea/delegate/battle/BattleState.java

```diff
@@ -5,10 +5,16 @@
 import games.strategy.engine.data.Territory;
 import games.strategy.engine.data.Unit;
 import java.util.Collection;
+import java.util.EnumSet;
 
 /** Exposes the battle state and allows updates to it */
 public interface BattleState {
 
+  enum Side {
```

Comment:
This was added to reduce `code complexity`.  I'm not exactly sure if this is the best name for it or if it should stay.